src/services/TemplatesService.js

```diff
@@ -0,0 +1,35 @@
+import db from '../db';
+
+const checkconflicts = (id, tenant) => new Promise((resolve, reject) => {
+  try {
+    let query;
+    if (id !== undefined) {
+      query = `select a.template_id, b.template_id as conflict from ${tenant}.attrs a \
```

Comment:
Don't forget to change the schema path before executing the SQL statement. I think there's a good example of how to do that in DeviceManager.

Reply:
[Here](https://github.com/dojot/device-manager/blob/9a5f62f7468909d292a79247cc286365778163b5/DeviceManager/TenancyManager.py#L70)

Reply:
I did some tests, and it is not necessary, because i'm using "schema.table". It already chage the schema automatically.

Reply:
Ok!